### ISSUE TITLE
about the data

### ISSUE THREAD

**Issue by NadiyaC249** [plmoer/skull_stripping_code_SR#3]
hello    really  a  good  work   ，can  i  ask   where did you get the data？ is it private？

**Comment by plmoer** [comment]
Yes, the training data is private from the University of Pittsburgh Medical Center.
